src/app/f2f/controllers/checkDetails.js

```diff
@@ -94,45 +95,50 @@ class CheckDetailsController extends DateController {
           break;
         }
         case APP.PHOTO_ID_OPTIONS.NON_UK_PASSPORT: {
+          idHasExpiryDate = req.form.values.idHasExpiryDate
```

Comment:
[nitpick] we have some inconsistencies around ; in this file, does running the linter pick it up?

Reply:
Linter doesn't pick up anything locally or in the GHA but am happy to sort out, what inconsistencies in particular?